packs/spell-effects/spell-effect-albatross-curse-failure.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "Y8OGpgAEgISpTwYd",
+    "img": "icons/creatures/birds/dove-pigeon-flying-white.webp",
+    "name": "Spell Effect: Albatross Curse (Failure)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p>The target must roll twice and take the worse result on their next Will save, after which the effect ends.</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse]</p>\n<p>The target must roll twice and take the worse result on their next Will save.</p>"
```

Reply:
Ok, changed.

But it would be nice to amend the style guide to say that effect descriptions should be re-worded to always refer to the character the effect is on in the 2nd person.  The existing effects in the system are not done this way, so anyone trying to match the existing style would not do this.  Rather, all, or nearly all, existing effects follow the general principle in the style guide, "For entry into the system we try to match Paizo's printed style where possible," and refer to the target of an effect in the 3rd person unless it is a self applied effect, which uses 2nd person.